Userland/Libraries/LibWeb/Layout/BlockFormattingContext.h

```diff
@@ -39,7 +39,7 @@ class BlockFormattingContext : public FormattingContext {
 
     void add_absolutely_positioned_box(Box const& box) { m_absolutely_positioned_boxes.append(box); }
 
-    SpaceUsedByFloats space_used_by_floats(CSSPixels y) const;
+    SpaceUsedAndEdgeForFloats space_used_by_floats(CSSPixels y) const;
```

Comment:
It's weird that `space_used_by_floats` no longer returns a `SpaceUsedByFloats` despite `SpaceUsedByFloats` still being a type.